elements/rh-card/rh-card.ts

```diff
@@ -9,12 +9,54 @@ import { colorContextProvider, type ColorPalette } from '../../lib/context/color
 import styles from './rh-card.css';
 /**
  * Card
- * @slot header
- *       If this slot is used, we expect a heading level tag (h1, h2, h3, h4, h5, h6).
- *       An icon, svg, or use of the icon component are also valid in this region.
- * @slot - Any content that is not designated for the header or footer slot, will go to this slot.
- * @slot footer
- *       Use this slot for anything that you want to be stuck to the base of the card.
+ * @slot        header
+ *              If this slot is used, we expect a heading level tag (h1, h2, h3, h4, h5, h6).
+ *              An icon, svg, or use of the icon component are also valid in this region.
+ * @slot        Any content that is not designated for the header or footer slot, will go to this slot.
+ * @slot        footer
+ *              Use this slot for anything that you want to be stuck to the base of the card.
+ * @cssprop     {<length>|<percentage>} --rh-border-width-lg
+ *              Border radius for card
+ *              {@default `3px`}
+ * @cssprop     {<color>} --rh-color-border-subtle-on-dark
+ *              Border color for card on dark background
+ *              {@default `#707070`}
+ * @cssprop     {<color>} --rh-color-border-subtle-on-light
+ *              Border color for card on light background
+ *              {@default `#c7c7c7`}
+ * @cssprop     {<color>} --rh-color-surface-darkest
+ *              Background color for card on dark backgrounds
+ *              {@default `#151515`}
+ * @cssprop     {<color>} --rh-color-surface-lightest
+ *              Background color for card on light backgrounds
+ *              {@default `#ffffff`}
+ * @cssprop     {<color>} --rh-color-text-primary-on-dark
+ *              Text color for card on dark background
+ *              {@default `#ffffff`}
+ * @cssprop     {<color>} --rh-color-text-primary-on-light
+ *              Text color for card on light background
+ *              {@default `#151515`}
+ * @cssprop     {<color>} --rh-font-family-body-text
```

Comment:
system tokens should only have names
https://github.com/RedHat-UX/red-hat-design-system/wiki/Docs-Cleanup-Q2-2023-Checklist